Fitness/Controllers/EndOnboardingLetterViewController.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  EndOnboardingLetterViewController.swift
+//  Fitness
+//
+//  Created by Phillip OReggio on 3/21/19.
+//  Copyright © 2019 Keivan Shahida. All rights reserved.
+//
+
+import UIKit
+
+class EndOnboardingLetterViewController: UIViewController {
```

Comment:
Is there any reason we have this file? If we don't use it we should just delete it

Reply:
We didn't have a finalized Onboarding Letter view at the time, so I was making what was there at the time (meant to delete this before, oops)